requirements.txt

```diff
@@ -3,3 +3,6 @@ fastapi==0.111.0
 loguru==0.7.2
 pydantic==2.7.1
 Requests==2.32.2
+flake8==7.1.1
+pre-commit=3.8.0
+black=24.8.0
```

Comment:
Should this be in requirements-dev.txt I don't think this is needed in production for example https://github.com/Ensembl/ensembl-web-tools-api/blob/dev/requirements-dev.txt